src/pages/komodo-defi-framework/api/v20/withdraw/index.mdx

```diff
@@ -561,6 +562,75 @@ This method generates a raw transaction which should then be broadcast using [se
   You can see the memo is included on the [block explorer](https://irishub.iobscan.io/#/txs/E00982A2A8442D7140916A34E29E287A0B1CBB4B38940372D1966BA7ACDE5BD6)
 </CollapsibleSection>
 
+##### Withdraw Tendermint IBC coins
+
+Using the Cosmos [Ecosystem IBC protocol](https://everstake.one/blog/cosmos-ibc-breaking-down-the-walls-between-blockchains), you can withdraw coins from one chain to another. This example shows a withdrawal from Cosmos to Osmosis.
+
+<CodeGroup title="Withdraw" tag="POST" label="withdraw" mm2MethodDecorate="true">
+  ```json
+  {
+    "mmrpc": "2.0",
+    "userpass": "RPC_UserP@SSW0RD",
+    "method": "withdraw",
+    "params": {
+      "coin": "ATOM",
+      "to": "osmo16drqvl3u8sukfsu4lm3qsk28jr3fahjac4rdw4",
+      "amount": 0.1,
+      "memo": "In the blackest of your moments, wait with no fear.",
+      "ibc_source_channel": "channel-141",
```

Comment:
It's better (and safer) to add a note that this channel value is only used for demonstration purposes and they should refer to https://ibc.iobscan.io/channels for getting a real channel id.

Reply:
This should be fixed now in https://7981a20e.komodo-docs.pages.dev/en/docs/